### DISCUSSION TITLE
Discord server ban appeal

### DISCUSSION THREAD

**Discussion by THCgm** [ldtteam/minecolonies#9939] [General]
Hey is there any way to appeal a ban on the minecolonies discord server?

**Comment by Thodor12** [comment]
You have to contact one of our moderators on Discord.
- motionlesstrain
- shindria.

**Reply by THCgm** [reply]
Alr I'll check em out

**Reply by MotionlessTrain** [reply]
Can you give a brief summary?

**Reply by THCgm** [reply]
Of what exactly?

**Reply by MotionlessTrain** [reply]
Why you were banned and what changed so you would be allowed back on the server again

**Reply by THCgm** [reply]
Most likely got banned for being annoying I don't remember anymore it's been some time since that

**Reply by Raycoms** [reply]
What is your discord account name for starts maybe?

**Reply by THCgm** [reply]
eripper

**Reply by MotionlessTrain** [reply]
Appeal approved: you have been unbanned
